src/qpmodel.jl

```diff
@@ -50,7 +50,7 @@ function QuadraticModel(c :: AbstractVector{T},
 end
 
 function QuadraticModel(c :: AbstractVector{T}, H :: AbstractMatrix;
-                        A :: AbstractMatrix=zeros(T, 0,length(c)),
+                        A :: AbstractMatrix=zeros(T, 0, length(c)),
                         lcon :: AbstractVector=zeros(T, 0), ucon :: AbstractVector=zeros(T, 0),
```

Comment:
Sorry for being annoying. There are a few other spaces missing around `=` (to be consistent everywhere).